index.html

```diff
@@ -29,196 +29,199 @@
+            <li class="can"><b>CANOPY</b></li>
+            <li class="dev"><b> Back End Dev</b></li>
+            <li class="dev"><b> 2015</b></li>
+          </ul>
+          <p class="presentation">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="languages">
+            <ul>
+              <li class="lang">HTML</li>
+              <li class="lang">CSS</li>
+              <li class="lang">JavaScript</li>
+            </ul>
           </div>
-          <li class="list">Frameworks <i class="fa fa-angle-right"></i></li>
-          <hr />
-          <li class="list">Skills <i class="fa fa-angle-right"></i></li>
-          <hr />
-        </ul>
-      </div>
+          <button class="button">See project</button>
+        </div>
+      </section>
+      <section class="works" id="section4">
+        <img class="photo" id="photo4" src="images/printing project.png" />
+        <div class="part4">
+          <h1 class="title">
+            Multi-Post <br />
+            Stories
+          </h1>
+          <ul class="canopy">
+            <li class="can"><b>CANOPY</b></li>
+            <li class="dev"><b>Back End Dev</b></li>
+            <li class="dev"><b>2015</b></li>
+          </ul>
+          <p class="presentation">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="languages">
+            <ul>
+              <li class="lang">HTML</li>
+              <li class="lang">CSS</li>
+              <li class="lang">JavaScript</li>
+            </ul>
+          </div>
+          <button class="button">See project</button>
+        </div>
+      </section>
     </section>
+    <div class="contentAndLangs">
+      <section class="about">
+        <div class="content" id="content2">
+          <h1>About Myself</h1>
+          <p class="presentation">
+            Hello!I'm a software developer and I can help you build a product ,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hesitate to contact me.
+          </p>
+          <p class="connect">LET'S CONNECT</p>
+          <div class="icons">
+            <ul>
+              <li>
+                <a href=""><i class="fa fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-linkedin"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-medium"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-github"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-angellist"></i></a>
+              </li>
+            </ul>
+          </div>
+          <button class="button">Get my resume</button>
+        </div>
```

Comment:
I suggest you use a `section` tag for the main container as I said in the previous comment, to make your project more professional.

Kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

>### Your Design
>
><img width="1375" alt="Screenshot 2023-05-05 at 1 56 02 PM" src="https://user-images.githubusercontent.com/91237525/236535443-a4fea468-3648-476e-856c-5ee1486a3031.png">
>
>### Te Figma design
>
><img width="583" alt="Screenshot 2023-05-05 at 2 10 43 PM" src="https://user-images.githubusercontent.com/91237525/236535596-bcf6d6fe-b5bf-4fce-b5ee-b54c1e8203e5.png">